.github/workflows/tests.yaml

```diff
@@ -214,11 +214,56 @@ jobs:
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
 
+  tests-defects:
```

Comment:
FYI: I added this to the GitHub Action workflows. You'll see that there is now a dedicated `tests-defects` workflow that will test all codes with the name "defects" in them. The reason I have this as a separate test suite is that there are additional dependencies your code requires, so I have it in its own little environment. You wouldn't have been expected to do this or know to do this, but I'm letting you know more just in case you're curious.